keras-to-hls/keras-to-hls.py

```diff
@@ -111,17 +111,31 @@ def main():
         for config,config_value in keras_layer["config"].items():
             if(config=="activation"):
                 layer['activation']=config_value
+            if(config=="epsilon"):
+                layer['epsilon']=config_value	
             #if(config=="units"):
                 #print("PARSED NUM OF NODES",config_value)
 
+        
         #Translate weights and biases from h5 file
-        found_weights = h5File[layer['name']].visit(find_kernel_in_h5)
-        weights = h5File['/{}/{}'.format(layer['name'],found_weights)][()]
-        found_bias = h5File[layer['name']].visit(find_bias_in_h5)
-        biases = h5File['/{}/{}'.format(layer['name'],found_bias)][()]
-        cur_n_zeros = print_array_to_cpp("w{}".format(layer_counter), weights, yamlConfig['OutputDir'])
-        print_array_to_cpp("b{}".format(layer_counter), biases, yamlConfig['OutputDir'])
-        layer['weights_n_zeros'] = cur_n_zeros 
+        if layer['class_name'] != 'BatchNormalization' and layer['class_name'] != 'Activation':
+         found_weights = h5File[layer['name']].visit(find_kernel_in_h5)
+         weights = h5File['/{}/{}'.format(layer['name'],found_weights)][()]
+         found_bias = h5File[layer['name']].visit(find_bias_in_h5)
+         biases = h5File['/{}/{}'.format(layer['name'],found_bias)][()]
+         cur_n_zeros = print_array_to_cpp("w{}".format(layer_counter), weights, yamlConfig['OutputDir'])
+         print_array_to_cpp("b{}".format(layer_counter), biases, yamlConfig['OutputDir'])
+         layer['weights_n_zeros'] = cur_n_zeros 
+        elif layer['class_name'] == 'BatchNormalization':
+         beta = h5File['/{}/{}/beta:0'.format(layer['name'],layer['name'])][()]
```

Comment:
For extracting beta and gamma, can you update this to the approach of the weights and bias above?  I found that this code does not work for this example `/afs/cern.ch/user/k/kreis/public/hls4ml/walter.tgz`